connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Loggers.java

```diff
@@ -179,16 +176,17 @@ private synchronized List<org.apache.logging.log4j.Logger> loggers(String namesp
 
     // visible for testing
     org.apache.logging.log4j.Logger lookupLogger(String logger) {
-        return LogManager.getLogger(logger);
+        return LogManager.getLogger(isValidRootLoggerName(logger) ? LogManager.ROOT_LOGGER_NAME : logger);
     }
 
     List<org.apache.logging.log4j.Logger> currentLoggers() {
         LoggerContext context = (LoggerContext) LogManager.getContext(false);
-        Collection<LoggerConfig> loggerConfigs = context.getConfiguration().getLoggers().values();
-        return loggerConfigs.stream()
-            .map(LoggerConfig::getName)
-            .distinct()
-            .map(LogManager::getLogger)
+        // Make sure root logger has been initialized
```

Comment:
Maybe we can collect loggers from configuration as well that the root logger is must included.
```java
        var results = new HashMap<String, org.apache.logging.log4j.Logger>();
        context.getConfiguration().getLoggers().forEach((name, logger) -> results.put(name, LogManager.getLogger(name)));
        context.getLoggerRegistry().getLoggers().forEach(logger -> results.put(logger.getName(), logger));
        return results.values();
```